### ISSUE TITLE
Split XandY into training and testing data

### ISSUE THREAD

**Issue by chadfurman** [clevertech/ml-stack-answers#7]
# Splitting the dataset into the Training set and Test set
from sklearn.cross_validation import train_test_split
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size = 1/3, random_state = 0)
